meinberlin/apps/contrib/templates/meinberlin_contrib/item_detail.html

```diff
@@ -127,6 +127,56 @@ <h2 class="moderator-statement__title">{% trans 'Official Statement' %}</h2>
 
     {% block vote_button %}{% endblock %}
 
+    {% block moderation_info %}
+        {% if is_moderator and module.blueprint_type in 'PB3,PB2,PB1' %}
+        <div class="offset-lg-3 col-lg-6">
+            <div class="moderation-info__accordion">
+                <details class="accordion" open>
+                    <summary class="moderation-info__accordion-title">
+                        <h2 class="moderation-info__title">{% translate 'Moderation' %}</h2>
+                        <i class="fa fa-chevron-down" aria-hidden="true"></i>
+                    </summary>
+                    {% if object.module.moderationtask_set or object.remark.remark %}
+                    <div class="moderation-info__accordion-body">
+                        {% if object.module.moderationtask_set %}
+                            <h3 class="moderation-info__section">{% translate 'Moderation tasks' %}</h3>
+                            <ul class="u-list-reset">
+                            {% for task in object.module.moderationtask_set.all %}
+                                <li class="moderation-info__list-item">
+                                {% if task in object.completed_tasks.all %}
+                                    <i class="fas fa-circle-check u-success" aria-role="img" aria-label="{% translate 'Done' %}"></i>
+                                {% else %}
+                                    <i class="far fa-circle u-muted" aria-role="img" aria-label="{% translate 'Open' %}"></i>
+                                {% endif %}
+                                {{ task.name }}
+                                </li>
+                            {% endfor %}
+                            </ul>
+                        {% endif %}
+                        {% if object.remark.remark %}
+                            <h3 class="moderation-info__section">{% translate 'Remark' %}</h3>
+                            <p>
+                                {{ object.remark.remark }}
+                            </p>
+                        {% endif %}
+                    </div>
+                    {% endif %}
+                </details>
+                {% get_item_url object 'moderate' False as moderate_url %}
+                {% if moderate_url %}
+                    <a
+                        href="{{ moderate_url }}"
+                        class="btn btn--light btn--small moderation-info__btn"
+                        data-embed-target="external">
+                            <i class="fas fa-pencil" aria-hidden="true"></i>
+                            {% translate 'Edit' %}
```

Comment:
@phillimorland Is that enough context for a11y or should we have sth. saying that it's editing the moderator-stuff? (Called "moderate" in the dropdwon above)

Reply:
@fuzzylogic2000 ok we should definitely have consistent naming I think and also we should maybe use a heading and section tag to announce the comments as it is a new section the having an edit button at end of a section is not confusing.